Automattic/wp-calypso
# Launchpad: Add components for newsletter Launchpads

Related to #79635

## Proposed Changes

* Add Launchpad components for the Newsletter Free and Newsletter Paid launchpads.

## Testing Instructions



* Apply https://github.com/Automattic/jetpack/pull/32067 to your sandbox and sandbox `public-api.wordpress.com`
* Enable the filters for free and paid newsletter Launchpads in your `0-sandbox.php`:

```
add_filter( 'wpcom_launchpad_intent_free_newsletter_enabled', '__return_true' );
add_filter( 'wpcom_launchpad_intent_paid_newsletter_enabled', '__return_true' );
```

* Create a new site from `/setup/newsletter` with the Free intent.
* You should see the corresponding task list on the My Home screen:

<img width="1086" alt="Screenshot 2023-07-25 at 2 15 41 PM" src="https://github.com/Automattic/wp-calypso/assets/2124984/8868ae4a-ac13-4d6a-a1d8-e39f03bafbbc">

* Because the task lists are currently the same, you can verify you're seeing the correct view by looking at the `layout` endpoint in the Network tab:

<img width="886" alt="Screenshot 2023-07-25 at 2 14 09 PM" src="https://github.com/Automattic/wp-calypso/assets/2124984/ea4978c1-2ea1-4282-abd2-5c801d4052a2">

* Create a new site from `/setup/newsletter` with the Paid intent and perform the same check of the `layout` endpoint; it should reference the `paid` newsletter view:

<img width="875" alt="Screenshot 2023-07-25 at 2 13 53 PM" src="https://github.com/Automattic/wp-calypso/assets/2124984/1a33a1a6-c9dd-444f-b0dd-7c364d40b766">

## Pre-merge Checklist
